sql_generators/terms_of_use/templates/terms_of_use_messages_v1/query.sql.jinja

```diff
@@ -0,0 +1,35 @@
+-- {{ header }}
+SELECT
+  submission_timestamp,
+  client_info.client_id,
+  sample_id,
+  metrics.uuid.messaging_system_client_id AS legacy_telemetry_client_id,
+  metrics.text2.messaging_system_message_id,
+  metrics.string.messaging_system_event,
+  metrics.string.messaging_system_event_source,
+  -- the events for the infobar are different than the events for the modal - this should standardize the modal events so they look like the infobar events
+  CASE
+    -- these are the infobar events or the events for which the infobar and modal are the same - treat them as-is
+    WHEN metrics.string.messaging_system_event IN ("IMPRESSION", "CLICK_PRIMARY_BUTTON", "DISMISSED", "CLICK_SECONDARY_BUTTON") THEN metrics.string.messaging_system_event
+    -- for the modal the modal DISMISS == DISMISSED
+    WHEN metrics.string.messaging_system_event = "DISMISS" THEN "DISMISSED"
+    WHEN metrics.string.messaging_system_event_source = "primary_button" THEN "CLICK_PRIMARY_BUTTON"
+    WHEN metrics.string.messaging_system_event_source = "secondary_button" THEN "CLICK_SECONDARY_BUTTON"
+    -- otherwise, its a privacy notice or other button click
+    ELSE UPPER(REPLACE(metrics.string.messaging_system_event_source, "-", "_")
+  ) END AS normalized_event,
+  -- standardize the surface of the message. there are two message_ids for the blue infobar - collapse them.
```

Comment:
```suggestion
  -- standardize the surface of the message. 
```

deleting part of the comment was left over from a previous version of this query